beacon/validator/src/test/java/tech/pegasys/teku/validator/coordinator/BlockOperationSelectorFactoryTest.java

```diff
@@ -497,23 +505,95 @@
         blobsBundle);
 
     final List<BlobSidecarOld> blobSidecars =
-        safeJoin(factory.createBlobSidecarsSelector().apply(block));
-
-    assertThat(blobSidecars)
-        .hasSize(blobsBundle.getNumberOfBlobs())
-        .first()
-        .satisfies(
-            // assert on one of the sidecars
-            blobSidecar -> {
-              assertThat(blobSidecar.getBlockRoot()).isEqualTo(block.getRoot());
-              assertThat(blobSidecar.getBlockParentRoot()).isEqualTo(block.getParentRoot());
-              assertThat(blobSidecar.getIndex()).isEqualTo(UInt64.ZERO);
-              assertThat(blobSidecar.getSlot()).isEqualTo(block.getSlot());
-              assertThat(blobSidecar.getProposerIndex()).isEqualTo(block.getProposerIndex());
-              assertThat(blobSidecar.getBlob()).isEqualTo(blobsBundle.getBlobs().get(0));
-              assertThat(blobSidecar.getKZGCommitment())
-                  .isEqualTo(blobsBundle.getCommitments().get(0));
-              assertThat(blobSidecar.getKZGProof()).isEqualTo(blobsBundle.getProofs().get(0));
+        safeJoin(factory.createBlobSidecarsSelectorOld().apply(block));
+
+    assertThat(blobSidecars).isEmpty();
```

Comment:
So we have this covered in new tests, just keeping this to be sure we have not broken anything old, right? If so I'd add something like `@Deprecated(since="with BlobSidecarOld, covered for BlobSidecar")` or comment or whatever

Reply:
More so to remember to add unit tests when we delete this method. I added TODO.